utils/wikize_refs.py

```diff
@@ -54,12 +61,18 @@ def usage():
 
     ./wikize_refs.py -i foo.md
 
-...which also backs up the original file as foo.md~.  Use option -s to skip
-the backup on the input file in this case.
+...which also backs up the original file as foo.md~.  Use option
+-s to skip the backup on the input file in this case.
 
     ./wikize_refs.py --help
 
 ...prints command-line arguments and options.
+
+If no *destructive* options are used, the following sed pipe command
+should be able to take the wikized file and produce the original...
```

Comment:
Okay, this makes it clear this pertains the the **wikized file**, independent of if this was an in-place modification or not.  When not modifying in place (i.e., leaving out the option `-i`), the source file is never modified.  It might be good to make that clear because the option ` -i, --in-place` does say "(destructive)" and that is not applicable here as a "destructive" option this is talking about. Otherwise, this is kind of confusing.

Reply:
Ok I addressed this with some additional verbiage in the usage statement. I also changed `destructive` to `irreversible`.